psi4/src/core.cc

```diff
@@ -541,8 +541,8 @@ bool py_psi_set_local_option_int(std::string const& module, std::string const& k
     std::string nonconst_key = to_upper(key);
     Data& data = Process::environment.options[nonconst_key];
 
-    if (data.type() == "double" && specifies_convergence(nonconst_key)) {
-        double val = pow(10.0, -value);
+    if (data.type() == "double") {
+        double val = (specifies_convergence(nonconst_key)) ? pow(10.0, -value) : double(value);
```

Comment:
better do the same for global option, too, https://github.com/psi4/psi4/blob/38f5c032bb9aa7aed1d873e76df6ca4d5e3e1a2f/psi4/src/core.cc#L581. then lgtm.

Reply:
Done.